app/webpacker/packs/application.js

```diff
@@ -1,11 +1,23 @@
 import "../styles/application.scss";
+import "accessible-autocomplete/dist/accessible-autocomplete.min.css";
 import { initAll } from "govuk-frontend";
 import initLocationsMap from "scripts/locations-map";
 import backLink from "scripts/back-link";
+import initAutocomplete from "scripts/autocomplete";
 
 initAll();
 
 window.initLocationsMap = initLocationsMap;
 
 const $backLink = document.querySelector('[data-module="back-link"]');
 new backLink($backLink).init();
+
+try {
+  const $autocomplete = document.getElementById("provider-autocomplete");
+  const $input = document.getElementById("query");
+  if ($autocomplete) {
+    initAutocomplete($autocomplete, $input);
```

Comment:
Would of thought this file would only contain just a call to initAutocomplete.

And initAutocomplete will contain the try catch along with the init code etc.

Reply:
@defong - looks like you have some refactoring to do!
https://github.com/DFE-Digital/search-and-compare-ui/blob/5cbef895923a452c7f20914e9c4acadba597f6eb/src/Assets/app.js#L40

Reply:
you copying bugs/defects again